UNRELEASED.md

```diff
@@ -1,3 +1,4 @@
-**Fixed**
```

Comment:
This was probably removed by accident? 😄

Reply:
I'm not sure about this, but when I contributed to the PR before, the UNRELEASED file was always empty, so I would just commit my changes. 
But this time I pulled main and found it had content in it, so I deleted it and added my own changes. Sorry about this, I will revert it back.

Reply:
It's probably because some PR was merged after you started the contribution 🙂 No problem!

Reply:
Have updated, thanks for your review!